README.md

```diff
@@ -432,3 +433,24 @@ If you have an idea for a new feature, don't hesitate to share it on the [issues
 <!-- prettier-ignore-end -->
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
+
+
+### Tips
+
+1. When running PySR with `julia 1.11`, the process seems to run into a memory leak bug that halts the execution by exceeding the available memory. Forcing the version `1.10` can help avoiding such problem:
```

Comment:
This memory leak is a Julia bug and will be fixed once 1.11.3 is released (as well as 1.10.8 hopefully) – https://github.com/JuliaLang/julia/pull/56801. So we probably don't need to provide this guidance as it will only be temporary until the new Julia is out?

Reply:
then maybe put as a highlighted issue (same as the "have you used PySR in your paper") while this is not fixed